synapse/storage/databases/main/events.py

```diff
@@ -1,6 +1,6 @@
 # Copyright 2014-2016 OpenMarket Ltd
 # Copyright 2018-2019 New Vector Ltd
```

Comment:
I believe we're supposed to combine all these (OM and NVL transferred the copyright to m.org)

```suggestion
```

Reply:
I think the latest I've seen is that we're not supposed to and we should just be adding to it?

Reply:
apologies, have reread the wall of text on the wiki about it and sounds like you're right.

Reply:
No problem! Seems guidance on this has flipped back and forth a few times!